website/static/js/registrationUtils.js

```diff
@@ -200,6 +245,7 @@ Question.prototype.addComment = function() {
     var comment = new Comment({
         value: self.nextComment()
     });
+	comment.seenBy.push(currentUser.fullname);
```

Comment:
@barbour-em we never want to rely on mutable fields to refer to users (i.e. user.fullname). 

Use the user's id instead.